crates/bevy_ui/src/flex/mod.rs

```diff
@@ -1,20 +1,23 @@
 mod convert;
```

Comment:
This impl breaks scale factor support. We'll need to fix that before merging.

Reply:
I forgot to mention it, sorry.
I'm not familiar with this topic, is it required to round the translations at all?
If it is we would need to cache the scale factor for every node.
If it's not we could just disable the final rounding in stretch and do all layout calculations in logical pixels, which is very simple.

Reply:
Yeah I think ideally we would use logical pixels to keep the code simple, but I do think stretch rounds on our behalf,  which would break things when we try to scale back up via the scale factor. We could either try to fix that in stretch or revert to the "physical pixel" approach used on `bevy_master`.

Reply:
I'd like to fix it in stretch, but it seems the maintainers are not responsive. How would you like to handle that?